server.go

```diff
@@ -4581,13 +4581,9 @@ func (s *server) OpenChannel(
 	// If the fee rate wasn't specified, then we'll use a default
 	// confirmation target.
 	if req.FundingFeePerKw == 0 {
-		estimator := s.cc.FeeEstimator
-		feeRate, err := estimator.EstimateFeePerKW(6)
-		if err != nil {
-			req.Err <- err
-			return req.Updates, req.Err
-		}
-		req.FundingFeePerKw = feeRate
+		req.Err <- fmt.Errorf("no FundingFeePerKw specified for " +
+			"the channel opening transaction")
```

Comment:
why do we change this? I think initially I did make it mandatory to include a conf target or feerate, then #8693 was added to revert it back - think we should move this change to a new commit in this PR if this is a bug fix.

Reply:
ohh ok, reverting it back at keeping it as it was.

Reply:
yes I think we can return an error here because we already do this in the parsing function.

Reply:
moved it into a separate commit, tho I want to mention this is not a bug fix, but just unifying the place where we inject a default setting.